Lib/defconQt/fontView.py

```diff
@@ -1519,6 +1517,7 @@ def __init__(self, parent=None):
         self.setLayout(mainLayout)
 
     def addGlyphSet(self, glyphNames=[], glyphSetName="New glyph set"):
+        glyphSetName = self.tr("New glyph set")
```

Comment:
Okay but then you need to stop overriding the function parameter, there's no point providing a parameter if it becomes a constant value.

````
[...] glyphSetName=None):
if glyphSetName is None:
    glyphSetName = self.tr("New glyph set")